lib/rack/session/encryptor.rb

```diff
@@ -302,7 +302,7 @@ def encrypt(message)
           encrypted_data = cipher.update(serialized_payload) << cipher.final
 
           data << cipher_iv
-          data << cipher.auth_tag(16)
```

Comment:
Is 16 the canonical default?

Reply:
Take what I'm about to say with a grain of salt. I'm **not** an expert in this field. It's my understanding as someone well-versed in applied cryptography.

The authentication tag represents a point in the Galois field GF(2^128), requiring 128 bits to specify. However, in some scenarios, it's considered OK to truncate the 128 bits to 96 bits. At the moment, Ruby's OpenSSL bindings return the entire auth tag - i.e., 128 bits, 16 bytes - but well, that can change. By being explicit, we are protecting ourselves against that scenario.